remoting/subsystem/src/main/java/org/jboss/as/remoting/LocalOutboundConnectionResourceDefinition.java

```diff
@@ -40,38 +44,39 @@ class LocalOutboundConnectionResourceDefinition extends AbstractOutboundConnecti
 
     static final PathElement ADDRESS = PathElement.pathElement(CommonAttributes.LOCAL_OUTBOUND_CONNECTION);
 
-    public static final SimpleAttributeDefinition OUTBOUND_SOCKET_BINDING_REF = new SimpleAttributeDefinitionBuilder(CommonAttributes.OUTBOUND_SOCKET_BINDING_REF, ModelType.STRING, false)
+    // TODO This is never used - why is it required?
```

Comment:
@pferraro @fl4via This dates to:

https://github.com/wildfly/wildfly-core/commit/7281132dee1f595f165720dcd4c91c7ee9dd7256

which came in as part of the big Elytron integration PR https://github.com/wildfly/wildfly-core/pull/2148

due to its incorporation of https://github.com/wildfly-security-incubator/wildfly-core/pull/37

I guess the local: protocol used to involve a host name in its URIs, but no longer does.

If this resource survives this attribute should be deprecated and made optional.

Reply:
It also shouldn't allow expressions :)